src/features/wallet/helpers/formatWalletBalance.ts

```diff
@@ -0,0 +1,28 @@
+// There are two different colours for the first 4 digits and last 4 digits of the number.
+// Hence we need to return an array of strings.
+// We want to limit the overall string length to ~8 digits.
+export default function formatWalletBalance(balance: string): string[] {
```

Comment:
Similar to my other comment above, I think this might be better suited for a different PR unless this deals with rewards (although, now that it's here, I think it's fine to keep in).